### PR TITLE
Update gradients_spheres.py

### PR DESCRIPTION
**Pull request by justbennet**
There were some indicators of inline code that were incorrect.  I swtiched the order of a couple of things to make more natural (to my ear) sounding English.  Corrected a minor typo at the top.  Change one instance of 'to' to 'two', which I believe was intended.  Removed trailing blank lines in a blind attempt to fix the unrendered
```
.. admonition:: Example source code
```
that appears as a code block in
http://nipy.org/dipy/examples_built/gradients_spheres.html#example-gradients-spheres

Apologies in advance if I am off-base with any of the changes.

### CODE REVIEW

File: doc/examples/gradients_spheres.py [hunk 1]
```diff
@@ -183,4 +183,3 @@
        anisotropic systems by magnetic resonance imaging, Magnetic Resonance
        in Medicine, vol 42, no 3, 515-525, 1999. 
 """
-
```
Comment (jchoude) [inline doc/examples/gradients_spheres.py]: I'm not sure removing the empty line will fix the rendering problem you noticed. Did you try rebuilding the example to see if it worked?

Comment (justbennet) [reply doc/examples/gradients_spheres.py]: No, I did not.  Hence calling it a 'blind attempt'.  I tried running `make html` and got an error at vtk, so I don't have the machinery to do a full make.

You're welcome to accept or decline any of the other changes and close the PR, but there appear now to be conflicts, as some of the sections I suggested changes to have been changed.

Please let me know if you would prefer that I not submit typo changes in the future without being able to run `make`.  If not, is it acceptable to submit them as issues?

Comment (jchoude) [reply doc/examples/gradients_spheres.py]: Hi,

yes, it seems that there are some conflicts. Most of them seem to be minor. Would you have the time to fix them?

Ideally, we prefer if people can run `make`, to ensure that the changes have the expected effect. It also helps the core maintainers (@skoudoro , @Garyfallidis ) if work is distributed. 

A PR is always better than an issue, but we understand in some cases if it is not possible.

I'm currently running the make on your branch to see if it works.

**Comment by justbennet** [comment]
I am not an MRI analyst, just a system administrator.  I ran across the typos while trying to figure out a problem for someone else.  I haven't got access to a machine with all the dependencies to set up the make.  The problem we were working on was on someone else's box.

The changes were all very minor -- the addition of quotes of one kind or another in a few places and deletion of an empty line.  Did that really break the make?  I looked at the Travis CI logs, and these appear to be the tracebacks at the breakage.

I am very curious to hear whether the `make` on just the documentation succeeds or not, and if now, which change broke it and where.

Thanks.  -- bennet

```
======================================================================
FAIL: dipy.reconst.tests.test_csdeconv.test_csdeconv
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/nipy/dipy/venv/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.5/site-packages/dipy/reconst/tests/test_csdeconv.py", line 233, in test_csdeconv
    assert_equal(len(w) > 0, False)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.5/site-packages/numpy/testing/nose_tools/utils.py", line 417, in assert_equal
    raise AssertionError(msg)
AssertionError: 
Items are not equal:
 ACTUAL: True
 DESIRED: False

----------------------------------------------------------------------
```



```
Collecting scipy

  Downloading scipy-0.19.1-cp34-cp34m-manylinux1_x86_64.whl (45.8MB)

    97% |███████████████████████████████▏| 44.6MB 45.0MB/s eta 0:00:01Exception:
Traceback (most recent call last):
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/_vendor/requests/packages/urllib3/response.py", line 232, in _error_catcher
    yield
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/_vendor/requests/packages/urllib3/response.py", line 314, in read
    data = self._fp.read(amt)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/_vendor/cachecontrol/filewrapper.py", line 60, in read
    data = self.__fp.read(amt)
  File "/opt/python/3.4.6/lib/python3.4/http/client.py", line 537, in read
    return super(HTTPResponse, self).read(amt)
  File "/opt/python/3.4.6/lib/python3.4/http/client.py", line 576, in readinto
    n = self.fp.readinto(b)
  File "/opt/python/3.4.6/lib/python3.4/socket.py", line 378, in readinto
    return self._sock.recv_into(b)
  File "/opt/python/3.4.6/lib/python3.4/ssl.py", line 748, in recv_into
    return self.read(nbytes, buffer)
  File "/opt/python/3.4.6/lib/python3.4/ssl.py", line 620, in read
    v = self._sslobj.read(len, buffer)
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/commands/install.py", line 335, in run
    wb.build(autobuilding=True)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/wheel.py", line 749, in build
    self.requirement_set.prepare_files(self.finder)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/req/req_set.py", line 380, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/req/req_set.py", line 620, in _prepare_file
    session=self.session, hashes=hashes)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/download.py", line 821, in unpack_url
    hashes=hashes
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/download.py", line 659, in unpack_http_url
    hashes)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/download.py", line 882, in _download_http_url
    _download_url(resp, link, content_file, hashes)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/download.py", line 603, in _download_url
    hashes.check_against_chunks(downloaded_chunks)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/utils/hashes.py", line 46, in check_against_chunks
    for chunk in chunks:
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/download.py", line 571, in written_chunks
    for chunk in chunks:
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/utils/ui.py", line 139, in iter
    for x in it:
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/download.py", line 560, in resp_read
    decode_content=False):
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/_vendor/requests/packages/urllib3/response.py", line 357, in stream
    data = self.read(amt=amt, decode_content=decode_content)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/_vendor/requests/packages/urllib3/response.py", line 324, in read
    flush_decoder = True
  File "/opt/python/3.4.6/lib/python3.4/contextlib.py", line 77, in __exit__
    self.gen.throw(type, value, traceback)
  File "/home/travis/build/nipy/dipy/venv/lib/python3.4/site-packages/pip/_vendor/requests/packages/urllib3/response.py", line 250, in _error_catcher
    raise ProtocolError('Connection broken: %r' % e, e)
pip._vendor.requests.packages.urllib3.exceptions.ProtocolError: ("Connection broken: ConnectionResetError(104, 'Connection reset by peer')", ConnectionResetError(104, 'Connection reset by peer'))
```

File: doc/examples/gradients_spheres.py (hunk 1 above)
Comment (justbennet) [reply doc/examples/gradients_spheres.py]: Did the make succeed or fail when you ran it?

**Comment by jchoude** [comment]
The failures do not seem to be related to your changes. 

The example built locally. There are some issues that do not seem to be related to your changes.

Do you think you would have a few minutes to rebase on master to remove the conflicts?

**Comment by justbennet** [comment]
It will be easier for me, actually, if you reject this PR.  I will reclone master and submit a new PR.

I presume that removing the trailing blank line did not fix the unrendered

.. admonition:: Example source code

that appears as a code block in

http://nipy.org/dipy/examples_built/gradients_spheres.html#example-gradients-spheres

So I will not include that in the new PR.  I think that will end up with a cleaner final repository for dipy.

Is that OK?

**Comment by jchoude** [comment]
Yes, it seems to have fixed it in my local test.

I'll decline the PR and let you create a new one. You can still remove the line as you did in this current PR.

Thanks for all of that!

**Pull request closed without merging**